src/promptflow/tests/sdk_cli_azure_test/unittests/test_flow_operations.py

```diff
@@ -22,19 +21,22 @@ def test_create_flow_with_invalid_parameters(self, pf):
 
         flow_source = flow_test_dir / "web_classification/"
         with pytest.raises(UserErrorException, match="Not a valid string"):
-            pf.flows.create_or_update(flow=flow_source, display_name=object())
+            pf.flows.create_or_update(flow=flow_source, display_name=False)
```

Comment:
why change object to False?
if display = False is a new case, you can create another sentence.

Reply:
Parameterize this test? pytest.mark.parametrize